ggplot_base_r_graph_analysis.Rmd

```diff
@@ -0,0 +1,115 @@
+---
```

Comment:
The whole YAML introduction should be cancelled including the --- lines . Instead, fix it with an introduction like this:
#Title of your project
**white line**
Your name
**white line ** 

Put the hashtag in front of the title of your project as it is marked above.